opentracing-shim/src/test/java/io/opentelemetry/opentracingshim/TracerShimTest.java

```diff
@@ -357,6 +357,12 @@ void extract_onlyBaggage() {
     assertThat(spanContextShim.getBaggage()).isEqualTo(baggage);
   }
 
+  @Test
+  void extract_null_spanctx() {
+    SpanContext spanContext = tracerShim.extract(Format.Builtin.TEXT_MAP, null);
+    assertThat(spanContext).isNull();
```

Comment:
Given that this PR only changes the log level, is this test trying to add missing coverage for existing behavior? Or verify logs are emitted at the intended level?

If the latter, I suggest we skip this. We do have test tooling to verify log messages (for [example](https://github.com/open-telemetry/opentelemetry-java/blob/2e8bf7f9a1150f4c92f296505032a544ffd2bd51/sdk-extensions/incubator/src/test/java/io/opentelemetry/sdk/extension/incubator/fileconfig/FileConfigurationCreateTest.java#L123-L127)), but asserting the log level seems rather specific.